src/main/java/baseball/Application.java

```diff
@@ -1,7 +1,145 @@
 package baseball;
 
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+
+        boolean continuePlaying = true;
+
+        System.out.println("숫자 야구 게임을 시작합니다.");
+
+        do{
+            play();
+            continuePlaying = askContinue();
+
+        }while(continuePlaying);
+
+    }
+
+    private static void play(){
+        List<Integer> randomNum = generateRandomNumber();
+        List<Integer> userNum;
+        String userInput;
+        boolean win = false;
+
+        do{
+            System.out.print("숫자를 입력해주세요:");
+            userInput = Console.readLine();
+            userNum = checkUserInput(userInput);
+
+            if(getGameResult(randomNum,userNum)){
+                win = true;
+                System.out.println("3개의 숫자를 모두 맞히셨습니다! 게임 종료");
+            }
+
+        }while(!win);
+    }
+
+    public static List<Integer> generateRandomNumber(){
+        List<Integer> computer = new ArrayList<>();
+        while(computer.size() < 3){
```

Comment:
```suggestion
        while(computer.size() < NUMBER_SIZE){
```
처럼 게임 규칙에 따라 변경될 수 있는 부분이라 생각돼 이 부분도 상수화해 관리하면 좋을 것 같습니다!